ios/swift/JitsiSDKTest/src/ViewController.swift

```diff
@@ -41,9 +42,17 @@ class ViewController: UIViewController {
         JitsiMeet.sharedInstance().defaultConferenceOptions = defaultOptions
     }
     
+    override func viewWillTransition(to size: CGSize,
+                                     with coordinator: UIViewControllerTransitionCoordinator) {
+        super.viewWillTransition(to: size, with: coordinator)
+
+        let rect = CGRect(origin: CGPoint.zero, size: size)
+        pipViewCoordinator?.resetBounds(bounds: rect)
+    }
+    
     @IBAction func openJitsiMeet(sender: Any?) {
         let room: String = roomName.text!
-        if(room.count < 1) {
+        guard room.count > 1 else {
```

Comment:
Question: while functionally equivalent, what's the reasoning behind this change? Is guard more idiomatic now in Swift?